content/kubernetes/active-active/global-config.md

```diff
@@ -21,22 +21,22 @@ The [REAADB API reference](https://github.com/RedisLabs/redis-enterprise-operato
 
 1. Edit or patch the REAADB custom resource with your global configuration changes.
```

Comment:
There are some formatting issues on this page. Most of the numbered steps aren't rendering correctly, and there are some formatting issues around the code block starting after "On each other participating cluster, check the secret status."